File: 3-Strings/1120-Revisao_de_Contrato-5/1120.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        while(true){
            string[] data=Console.ReadLine().Split(' ');
            string result = "";
            
            if(data[0] == "0" && data[1] == "0"){
                break;
            }
            
            
            foreach(char i in data[1]){
                if(Convert.ToString(i) != data[0]){
                    result+=Convert.ToString(i);
                }
            }

            result=result.TrimStart('0');
            
            if (string.IsNullOrEmpty(result))
            {
                Console.WriteLine(0);
            }
            else{
                Console.WriteLine(result);
            }
            
            
        }
    }
}

File: 3-Strings/1120-Revisao_de_Contrato-5/1120.py
while True:
    data = input().split(' ')
    if data == ['0', '0']:break
    digito=int(data[0])
    numero=list(data[1])
    while data[0] in numero:
        numero.remove(data[0])
    if numero == []:
        print(0)
    else:    
        result=''.join(numero)
        print(int(result))